nx_arangodb/classes/digraph.py

```diff
@@ -255,6 +269,50 @@ def add_node_override(self, node_for_adding, **attr):
 
         nx._clear_cache(self)
 
+    def add_nodes_from_override(self, nodes_for_adding, **attr):
+        for n in nodes_for_adding:
```

Comment:
Cant we optimize this and make use (if not complete, maybe partially) of `def add_node_override(self, node_for_adding, **attr):` ? 

At least from reading, `add_node_override` handles one node, while this one handles multiple handled with a loop. Should be doable.

Reply:
I did ponder this at first, but the original NetworkX `add_node` and `add_nodes_from` methods are implemented independently from each other:

1. https://github.com/networkx/networkx/blob/main/networkx/classes/graph.py#L522
2. https://github.com/networkx/networkx/blob/main/networkx/classes/graph.py#L571

So I figured that I keep the Monkey Patching to a minimum amount, and just keep everything else as-is.

However, I think we should revisit `add_nodes_from` & `add_edges_from` to make use of bulk insertion instead (so using `insert_many` or `import_bulk`)...

Reply:
Let's just create a low prio (opt) ticket for this, to not forget about it. Or alternatively solve this when it becomes a real world issue.